### DISCUSSION TITLE
MQTT Daten

### DISCUSSION THREAD

**Discussion by neolauren-cmd** [softwarecrash/Victron2MQTT#33] [General]
Hallo und vielen Dank für dieses Mega Projekt!

Ich habe noch eine Frage, wird es irgendwann noch möglich sein, die Ladezustände wie Bulk, Absorption und Float über MQTT zu empfangen? Das wäre für mich noch der berüchtigte I Punkt.

Danke und Gruß 
Neo

**Comment by softwarecrash** [comment]
alle daten die über VE.Direct ausgegeben werden sind im mqtt auch vorhanden.
siehe https://github.com/softwarecrash/Victron2MQTT/blob/main/Protocol/VE.Direct-Protocol-3.33.pdf
was dort nicht aufgeführt ist kann auch nicht ausgeben werden, zumindest nicht über die art wie victron2mqtt arbeitet

**Reply by neolauren-cmd** [reply]
Hab es auch gefunden. Wird zum Glück doch mit ausgegeben.
